beta-src/src/utils/drawArrow.ts

```diff
@@ -0,0 +1,52 @@
+export default function drawArrow(
+  /**
+   * color code passed from enum ArrowColors based on move type
+   */
+  arrowColor: string,
```

Comment:
This shouldn't be a prop, the correct color should be deduced based on the order type inside this function. Using the current theme and receiving a proper order type enum.

Reply:
I was under the assumption that this task was to take in two ids and a color and draw the line? Wouldnt the user of the arrow drawing function use the palette I created and pass that value to my function?